supabase/migrations/20240620042716_policy_add_tasks_permisions.sql

```diff
@@ -0,0 +1,15 @@
+alter table tasks enable row level security;
+
+create policy "anyone can view tasks"
+on tasks for select
+to authenticated, anon
+using ( true );
+
+create policy "customer user can create a new task"
+on tasks for insert
+to authenticated
+with check ( (select auth.uid()) = customer_id );
```

Comment:
A service provider user could create a task with this policy definition.

We'll investigate further in #38.